### PR TITLE
fixed is_dvr environment check

### PR DESCRIPTION
**Pull request by TregubovYury**
Fixed env check for DVR configurations.

### CODE REVIEW

File: mos_tests/conftest.py [hunk 1]
```diff
@@ -314,9 +314,8 @@ def is_dvr(env):
     """Env deployed with enabled distributed routers support"""
     controller = env.get_nodes_by_role('controller')[0]
     with env.get_ssh_to_node(controller.data['ip']) as remote:
-        with remote.open('/etc/neutron/plugin.ini') as f:
-            return get_config_option(
-                f, 'enable_distributed_routing', bool) is True
+        with remote.open('/etc/neutron/neutron.conf') as f:
+            return get_config_option(f, 'router_distributed', bool) is True
```
Comment (gdyuldin) [inline mos_tests/conftest.py]: `return get_config_option(f, 'router_distributed', bool)`

Comment (TimurNurlygayanov) [inline mos_tests/conftest.py]: +1

Comment (TregubovYury) [inline mos_tests/conftest.py]: Hmmmmmm.... should I change the same return for the is_l2pop and l3_ha then ?

Comment (TimurNurlygayanov) [inline mos_tests/conftest.py]: Yes please :)

**Pull request merged**